.travis.yml

```diff
@@ -1,8 +1,20 @@
 language: node_js
 
+services:
+  - docker
+
 node_js:
  - "8"
 
 install:
  - npm install
  - npm run-script build
+
+script:
+ - docker build -t dojot/iotagent-json:latest . -f ./docker/Dockerfile
+
+after_success:
+  - if [ "${TRAVIS_BRANCH}" == "cpqd_master" ]; then
```

Comment:
This will effectively prevent us from using travis to generate artifacts for tags.
On other repos, to keep travis' configurations smaller, we've used a helper [script](https://github.com/dojot/gui/blob/master/travis/publish.sh) to handle the container building part.

Reply:
I'll copy from there.